lib/matobj.gi

```diff
@@ -654,3 +658,14 @@ InstallMethod( IsEmptyMatrix,
   [ IsMatrixObj ],
   mat -> NrRows(mat) = 0 or NrCols(mat) = 0
 );
+
+
+#
+# Compatibility code: old variants of arguments (to become obsolete)
+#
+
+InstallOtherMethod( Randomize,
+    "backwards compatibility: swap arguments",
+    [ IsObject and IsMutable, IsRandomSource ],
+    { obj, rs } -> Randomize( rs, obj ) );
```

Comment:
This compatibility method ensures that existing code *using* `Randomize` will keep working, and can thus be adapted incrementally -- very good. However, existing *implementations* will need to be adapted, else they won't get called... I don't see a good alternative to this, anyway (other than installing another method which swaps the arguments in another order, and then perhaps also a horrible hack which prevents us from doing an infinite recursion in case no specialized method is provided). Luckily, only three packages provide such methods: `cvec` and `recog` (which I'll both adapt now, and make releases), and also `meataxe64`, but that's not yet deposited and I am sure @stevelinton will be happy to adapt it, resp. we can provide a PR. (And by adapting, I mean I'll install methods for both argument orders, this way the packages will work with both old and new GAP versions).